src/Sulu/Component/Content/Types/ResourceLocator/Mapper/PhpcrMapper.php

```diff
@@ -214,7 +214,7 @@ function($resourceLocator, NodeInterface $node) use (&$result) {
         \usort(
             $result,
             function(ResourceLocatorInformation $item1, ResourceLocatorInformation $item2) {
-                return $item1->getCreated() < $item2->getCreated();
+                return $item2->getCreated() <=> $item1->getCreated();
```

Comment:
looks likesomething we should test?

Reply:
okay result didn't change just the return type?

Reply:
yes only return type, it is still descending but with the spaceship operator I had to change the item2/item1 order